trade-finance-manager-ui/package.json

```diff
@@ -56,6 +56,7 @@
     "dotenv": "16.0.3",
     "express": "^4.18.2",
     "express-session": "1.17.3",
+    "express-xss-sanitizer": "^1.1.6",
```

Comment:
This is a wrapper around `sanitize-html`: https://www.npmjs.com/package/sanitize-html?activeTab=readme

Which has a default list of allowed tags (in the Default Options) section of that link. @abhi-markan, do you know what tags if any we should allow for these inputs?

Reply:
Hey @JennyHumphries no tags should be allowed. Thanks

Reply:
Great, thanks! The change to disallow all tags is in `middleware/xss-sanitizer.js`